config/locales/client.en.yml

```diff
@@ -2135,6 +2135,10 @@ en:
           one: You have selected <b>1</b> post.
           other: "You have selected <b>{{count}}</b> posts."
 
+      deleted_by_author:
```

Comment:
`post.deleted_by_author` is used by https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/models/post.js.es6

Should the client use `topic.deleted_by_author` as well? Otherwise `server.en.yml` would be a better place for this string.